cmd/status.go

```diff
@@ -64,14 +64,14 @@ type sections struct {
 }
 
 type statistic struct {
-	UsedSpace         uint64
-	AvailableSpace    uint64
-	UsedInodes        uint64
-	AvailableInodes   uint64
-	DeletedSliceCount int64 `json:",omitempty"`
-	DeletedSliceSize  int64 `json:",omitempty"`
-	DeletedFileCount  int64 `json:",omitempty"`
-	DeletedFileSize   int64 `json:",omitempty"`
+	UsedSpace               uint64
+	AvailableSpace          uint64
+	UsedInodes              uint64
+	AvailableInodes         uint64
+	TrashSliceCount         int64 `json:",omitempty"`
+	TrashSliceSize          int64 `json:",omitempty"`
+	DelayedDeletedFileCount int64 `json:",omitempty"`
+	DelayedDeletedFileSize  int64 `json:",omitempty"`
```

Comment:
remove `Delayed`

Reply:
I agree.

Reply:
need @davies to make the final call.

Reply:
To be clear, we should also can trash files and slices with ref < 0, then we will know:

used space = (normal space + trash space)
normal space = total of normal files
trash space =  total of trash files
pending space = total of pending files

object space = used slice (used by normal file, trash file or pending file) + unused space (trash slice + pending slice (refs <0) ) + leaked space (unknown)

Reply:
Fine, I'm going to rename current `DeletedFile` to `PendingDeletedFile`, then open another PR to scan `TrashFile` and `PendingDeletedSlice`.

Reply:
> Fine, I'm going to rename current `DeletedFile` to `PendingDeletedFile`, then open another PR to scan `TrashFile` and `PendingDeletedSlice`.

Renamed